pop-api/examples/nfts/lib.rs

```diff
@@ -40,34 +40,34 @@ mod pop_api_extension_demo {
 			receiver: AccountId,
 		) -> Result<(), ContractError> {
 			ink::env::debug_println!(
-				"Contract::mint_through_runtime: collection_id: {:?} item_id {:?} receiver: {:?}",
+				"Nfts::mint_through_runtime: collection_id: {:?} item_id {:?} receiver: {:?}",
```

Comment:
Since you changed `transfer_through_runtime` in the balance example, you should change the function name here from `mint_through_runtime` to `mint` and the corresponding log messages.